tests/docker-images/java-test-functions/pom.xml

```diff
@@ -82,24 +82,9 @@
                             </transformers>
                             <artifactSet>
                                 <includes>
-                                    <include>org.apache.pulsar:pulsar-client-original</include>
-                                    <include>org.apache.pulsar:pulsar-client-api</include>
-                                    <include>org.apache.pulsar:pulsar-client-admin-api</include>
-                                    <include>org.apache.pulsar:pulsar-functions-api-examples</include>
```

Comment:
Maybe the package `org.apache.pulsar:pulsar-functions-api-examples` is necessary, because the function API test examples all in this package.